CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+### Change Log
```

Comment:
Probably go with the format from http://keepachangelog.com/en/1.0.0/.

I think that's what the other changelogs are based on, right?